scripts/runtestsuite/runtestsuite.py

```diff
@@ -245,8 +293,18 @@ def __write_header(self, args):
                                        'script-version': VERSION,
                                        'appleseed-binary-path': args.tool_path,
                                        'max-abs-diff-allowed': VALUE_THRESHOLD,
-                                       'max-diff-comps-count-allowed': MAX_DIFFERING_COMPONENTS}))
+                                       'max-diff-comps-count-allowed': MAX_DIFFERING_COMPONENTS,
+                                       'git-hash': git_hash,
+                                       'git-title': git_title}))
         self.file.flush()
+    
+    def __write_stats(self, total_time, success_rate, failures, rendered_scenes):
+        success_rate = "{0}%".format(success_rate)
+        failures = "{0} out of {1} test scene(s)".format(failures, rendered_scenes)
+        self.file.write(self.__render(self.stats_template,
+                                       {'total-time': format_duration(total_time),
+                                        'success-rate': success_rate,
+                                        'failures': failures}))
```

Comment:
You can compute the success_rate here. Or, you can pass 3 count instead of 1 rate + 2 count, which is more consistent.

```suggestion
    def __write_stats(self, total_time, success_test_count, failed_test_count, total_test_count):
        success_rate = int(success_test_count / total_test_count * 100.0)
        
        total_time_text = format_duration(total_time)
        success_rate_text = "{0}%".format(success_rate)
        failures_text = "{0} out of {1} test scene(s)".format(failed_test_count, total_test_count)}))
        
        self.file.write(self.__render(self.stats_template,
                                       {'total-time': total_time_text,
                                        'success-rate': success_rate_text ,
                                        'failures': failures_text }))                                                                                
```

What do you think ?

Reply:
I think I mislead you in my last comment when suggesting to remove the suffix `_str`. My bad

Reply:
I like your suggestion below of passing the values around in a class but I think it would still be useful to have the `success rate` included, since the number is used more than once and the calculation is quite long (you also need to check for 0 rendered scenes to avoid a divide by 0).